.pre-commit-config.yaml.jinja

```
@@ -79,6 +79,7 @@ repos:
           - "@prettier/plugin-xml@{{ repo_rev.prettier_xml }}"
         args:
           - --plugin=@prettier/plugin-xml
+        files: \.xml$
```

I'm not sure, but shouldn't this be something like this?
```suggestion
        files: "\\.xml$"
```
This is following the same as in https://github.com/OCA/oca-addons-repo-template/pull/46/files#diff-643aa373eb3374e32d8b29a56a7d59d32f2f6bc86fcb5efd1d926f51b1364f17R29

---

Hi João
- on a yaml point of view `\.xml$` and `"\\.xml$"` are equivalent 
- jinja template evaluation doesn't mess with `\.xml$` so I don't see the need for what I perceive as the more complex form  `"\\.xml$"`
- in the end it sounds like a subjective preference; I don't think that the yaml parser of prettier supports enforcing one or the other form (see https://github.com/prettier/prettier/issues/5350) but you might want to create an issue at this repo level if you think it's worth :-)

---

Ok, I was just suggesting for a matter of consistency, but your approach seems perfectly fine :smiley: 
I thought the escaping character `\` was needed but indeed it isn't :sweat_smile: 
Thank you for the explanation :+1: